inference-engine/src/mkldnn_plugin/mkldnn_infer_request.cpp

```diff
@@ -57,8 +57,8 @@ void MKLDNNPlugin::MKLDNNInferRequest::pushInput(const std::string& inputName, I
 
 namespace {
 
-template <typename T>
-void copyToFloat(float* dst, const InferenceEngine::Blob* src) {
+template <typename T, typename DstT>
+void copyFrom(DstT* dst, const InferenceEngine::Blob* src) {
```

Comment:
Here we have mixed up order of template and function arguments. Looks like we need to swap `dst` and `src`

Reply:
will change